networking_cisco/plugins/cisco/service_plugins/cisco_router_gbp_plugin.py

```diff
@@ -0,0 +1,128 @@
+# Copyright 2016 Cisco Systems, Inc.  All rights reserved.
```

Comment:
this file is almost exactly the same as cisco_router_plugin.py file, why can't we just reuse cisco_router_plugin.py then?